account_move_batch_validate/models/account_move.py

```diff
@@ -2,13 +2,22 @@
 # Copyright 2014 Camptocamp SA, 2017 ACSONE
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
+from odoo import api, fields, models, _
 import logging
 
-from odoo import api, fields, models, _
-from odoo.addons.queue_job.job import job, Job
+_logger = logging.getLogger(__name__)
 
+try:
+    from odoo.addons.queue_job.job import job, Job
+except ImportError:
+    _logger.debug('Can not `import queue_job`.')
+    import functools
+
+    def empty_decorator_factory(*argv, **kwargs):
+        return functools.partial
+    job = empty_decorator_factory
+    Job = empty_decorator_factory
```

Comment:
`Job = empty_decorator_factory` is not required as it is only called in a method

Reply:
thanks